File: app_manager/appmanager/src/main/java/app/editors/manager/mvp/models/response/ResponseExplorer.java
package app.editors.manager.mvp.models.response;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import app.editors.manager.mvp.models.base.Base;
import app.editors.manager.mvp.models.explorer.Explorer;

public class ResponseExplorer extends Base {

    @SerializedName(KEY_RESPONSE)
    @Expose
    private Explorer response;

    public Explorer getResponse() {
        return response;
    }

    public void setResponse(Explorer response) {
        this.response = response;
    }

}

File: app_manager/core/src/main/java/app/documents/core/network/manager/models/response/ResponseExplorer.kt
package app.documents.core.network.manager.models.response

import app.documents.core.network.common.models.BaseResponse
import app.documents.core.network.manager.models.explorer.Explorer
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ResponseExplorer(
    @SerializedName(KEY_RESPONSE)
    @Expose
    var response: Explorer? = null
) : BaseResponse()